pkg/cmd/cli/cmd/edit.go

```diff
@@ -417,7 +417,7 @@ func applyPatch(delta *jsonmerge.Delta, info *resource.Info, version string) err
 // their updates were preserved.
 func preservedFile(err error, path string, out io.Writer) error {
 	if len(path) > 0 {
-		if _, err := os.Stat(path); !os.IsNotExist(err) {
+		if _, err = os.Stat(path); !os.IsNotExist(err) {
```

Comment:
I feel like this should result in a name change to `statErr` or something to be consistent with shadowing checks of other non-argument locals. @stevekuznetsov @liggitt thoughts?